turbo.json

```diff
@@ -5,7 +5,7 @@
   "tasks": {
     "build": {
       "dependsOn": ["^build"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**", "out/**"]
```

Comment:
If you change the output path from `apps/bundle-analyzer/out` to `apps/bundle-analyzer/dist` it would make it much easier and consistent